osu.Game.Rulesets.Osu/Edit/OsuHitObjectComposer.cs

```diff
@@ -42,10 +42,12 @@ protected override DrawableRuleset<OsuHitObject> CreateDrawableRuleset(Ruleset r
         };
 
         private readonly Bindable<TernaryState> distanceSnapToggle = new Bindable<TernaryState>();
+        private readonly Bindable<TernaryState> rectangularGridSnapToggle = new Bindable<TernaryState>();
 
         protected override IEnumerable<TernaryButton> CreateTernaryButtons() => base.CreateTernaryButtons().Concat(new[]
         {
-            new TernaryButton(distanceSnapToggle, "Distance Snap", () => new SpriteIcon { Icon = FontAwesome.Solid.Ruler })
+            new TernaryButton(distanceSnapToggle, "Distance Snap", () => new SpriteIcon { Icon = FontAwesome.Solid.Ruler }),
```

Comment:
Not sure this should be the home for these buttons (since they aren't ternary) but it'll do for now.